app.js

```diff
@@ -0,0 +1,82 @@
+'use strict';
+var express = require('express');
+var trucks = require('./trucks');
+var _ = require('underscore')
+var bodyParser = require('body-parser')
+var app = express();
+app.use(bodyParser.urlencoded({extended: true}));
+app.use(bodyParser.json());
+app.use(express.static('public'));
+
+app.get('/trucks', function (request, response) {
+  response.json(trucks.getTrucks());
+});
+
+app.post('/trucks', function (request, response){
+  var newTruck = request.body;
+  var foodTrucks = trucks.getTruck();
```

Comment:
This function shouldn't return anything if nothing was passed in. It's purpose is to return a truck based on a name passed in.